README.md

```diff
@@ -244,6 +244,36 @@ To register the Logback collector can be added to the root level like so:
 </configuration>
 ```
 
+#### Hibernate
+
+There is a collector for Hibernate which allows to collect metrics from one or more 
+`SessionFactory` instances. To use this collector add the following dependency:
+
+```xml
+<dependency>
```

Comment:
We haven't included this with the others, and I'd prefer not to have another place to manually update version numbers.